scripts/accumulator-server.py

```diff
@@ -207,16 +207,18 @@ def bad_response():
     r.data = '{"name":"ENTITY_NOT_FOUND","message":"The entity with the requested id [qa_name_01] was not found."}'
```

Comment:
A lot of diff noise...

Basically what I have done is to refactor notification storage code in record_request() and add a new function record_2871() prepared to return the ouout that Orions needs to reproduce the case described in the issue. I don't like too much this kind of ad hoc code, as long as we don't have too much exceptional cases and they are easily "plugable" in record_XXXX() functions.

NTC (just informative)